TasktrayUI/WindowsMediaToLiveMessenger/LiveMessengerWrapper.cs

```diff
@@ -63,17 +63,17 @@ public struct COPYDATASTRUCT
         /// <summary>
         /// This is the format of the message that 
         /// will be displayed on MSN. The values 
-        /// "{0}", "{1}" and "{3}", represent the
+        /// "{0}", "{1}" and "{2}", represent the
         /// song name, artist and album name
         /// respectively.
         /// </summary>
-        private string _format = "On Windows 10: ♪♫{1} - {0}♪♫";
-        private const string _formatDefault = "On Windows 10: ♪♫{1} - {0}♪♫";
+        private string _format = "{1} - {0}";
+        private const string _formatDefault = "{1} - {0}";
```

Comment:
Also, I do like having the "♪♫" being part of the default message. I know they look goofy but that's kinda the point of having them in the first place (plus you can always modify the message at runtime to remove them if you prefer). This is a big nitpick so please tell me what do you think and if you prefer to remove them then I am fine with you merging your PR as it is.

Reply:
I figured that haha, but thought it wasn't really useful, since MSN already have very similar characters at the beginning of the label, just not as a text but rather as an icon. They're goofy indeed though!
This is your project though, you might not agree with me and might really want to keep them. Feel free to ask me to put them back, of course!

Reply:
Well you are right that MSN already has them displayed with their own "emoticons". You can delete them then, it already looks good without them!